regression/cbmc/address_space_size_limit3/main.i

```diff
@@ -92,6 +92,11 @@ void f__L_0x3c6_0()
 
 int main()
 {
+  int unknown;
+  // Avoid constant propagation solving for pointer-offsets of esp,
+  // which prevent demonstrating the object-size limitation this test
+  // intends to exhibit:
+  esp -= (unknown & 1);
```

Comment:
This actually changes the semantics of the program? Maybe do `esp += (unknown & 1);` to counter that?